relayer/msgs.go

```diff
@@ -156,3 +161,106 @@ func (c *Chain) MsgTransfer(dst *PathEnd, amount sdk.Coin, dstAddr string,
 		timeoutTimestamp,
 	)
 }
+
+// CreateClient creates an sdk.Msg to update the client on src with consensus state from dst
+func (c *Chain) CreateClient(
```

Comment:
this is a nit, but we could order these logically like how the are in ibc core. The ordering is:
CreateClient
UpdateClient
ConnOpenInit
ConnOpenTry
...
ChanOpenInit
ChanOpenTry
...
ChanConfirmInit
...

it just makes it easier to navigate